### ISSUE TITLE
Template Heading showing only None, despite defined in templates 

### ISSUE THREAD

**Issue by aniketgm** [lumoe/obsidian-rollover-daily-todos#152]
### Issue:

I have a template file with headings defined, however, the headings are not shown in the Templates option. Only None shows. Not sure what am I missing or what the issue is ?

**Template File**
![image](https://github.com/user-attachments/assets/1596febb-c992-42a6-8a08-4198225db7be)

**Option showing only None**
![image](https://github.com/user-attachments/assets/ccded975-e09c-4bd5-89c4-1006e40da219)

### Assumption (based on observation):

If the template format e.g. `{{date}}` is an issue for parsing template headings. I would suggest, ignore such patterns, and grab only the english alphabets. This is a standard pattern which comes out of the box pattern with the core 'Template' plugin

**Comment by masenka31** [comment]
I was just searching if anyone else has the same problem... +1 on this. I am using Templater and my Daily Note template has just maybe a different format for the name? `YYYY-MM-DD dddd` Not sure what would be the problem, but I didn't yet have the time to look through to code to figure it out.

**Comment by masenka31** [comment]
For anyone that has this issue, there is a workaround. Open the Obsidian Console and change the setting for the value of the heading you want to use from your template:

```
const plugin = app.plugins.plugins["obsidian-rollover-daily-todos"]
plugin.settings.templateHeading = 'TODOs'
plugin.saveSettings()
```

When the `saveSettings()` is used, the setting seems to be permanent even after a restart of Obsidian. It will probably be changed on a plugin update, so just run it again and it should work.

Btw. the problem seems to be in the `obsidian-daily-notes-interface` library and it's `getDailyNoteSettings` function that cannot read the correct template file.

**Comment by mmanela** [comment]
I am hitting the same issue, I will test your solution @masenka31

**Comment by dizzi101** [comment]
I have a same issue but I am not familiar with coding so I am unable to test the suggestion by @masenka31 . I figured out how to open console - is it correct that I should go to Obsidian file menu and choose View-Toggle Developer Tools, then choose the tab  called "Sources" then finally from menu on the left choose the line plugin:obsidian-rollover-daily-todos; this opens up a page of code with ca. 1000 lines which presumably is the code for that plugin? 

However I am unable to find this specific text in the entire 1000+ line code: 
const plugin = app.plugins.plugins["obsidian-rollover-daily-todos"]
plugin.settings.templateHeading = 'TODOs'
plugin.saveSettings()

Either it is not there or I am in a completely wrong place. 

To summarise my situation: I have installed Rollover Daily Todos plugin which works, but it doesn't allow me to select a header from my daily note (created with periodic notes) into which to rollover the unticked items; just like in the original post by @aniketgm , I also get option "none" only. 

My template is very simple, starts like this

## ToDos

- [ ] to do 
- [ ] to do 


I would much appreciate someone to explain how to edit the code (how/where exactly to find that line) if this really fixes the problem! I really like this plugin and it would be great if it could start behaving as intended.

**Comment by masenka31** [comment]
Hi @dizzi101,

You just need to open the Obsidian Developer Console with `cmd + option + i`. As you open the console, you should be able to write to it. Just copy-paste the snippet and hit enter :)

<img width="701" height="641" alt="Image" src="https://github.com/user-attachments/assets/e06d6862-87b3-448e-b410-e3d0b0eaaae7" />

**Comment by dizzi101** [comment]
After a bit more fiddling this actually worked, so thanks a lot for your help

**Comment by 8petros** [comment]
How can I performance this in Android APP?
